twilight-model/src/gateway/event/mod.rs

```diff
@@ -457,14 +457,11 @@ mod tests {
     // requires a variable to be used in a function, so this is a false
     // positive.
     #[allow(dead_code)]
-    const EVENT_THRESHOLD: usize = 224;
+    const EVENT_THRESHOLD: usize = 256;
```

Comment:
Note to self: why not use <https://rust-lang.github.io/rust-clippy/master/index.html#/large_enum_variant>

Reply:
i think revising the enabled lints deserves its own issue, there probably are other useful lints we're missing, i could create that issue if yall agree, [this](https://gist.github.com/laralove143/1fab39b9f2c3f2760633cbfb0a8c1c8d) is the very restrictive set of lints i use for example, ofc that may be too strict for the collaborative nature of twilight but its an example of the kinds of lints we can make use of

Reply:
A tracking issue for lints would be great. There has been a lot of additions to Clippy in the last 2 years that we have not implemented

Reply:
done at #2350